docs/guides/security-guide.md

```diff
@@ -113,11 +100,11 @@ More info on these values can be found on [this gRPC library documentation page]
 
 Ports can be opened in insecure mode if needed. This is controlled
 by a flag that needs to be enabled at runtime. Change the
-`grpc_open_insecure_ports` value to `true` to open insecure ports.
+`grpc_open_insecure_mode` value to `true` to allow insecure communication
 Also, make sure `certs` directory is removed from the default location mentioned
 above.
 
-To launch infrap4d with insecure ports 9339 and 9559 open:
+To launch infrap4d in insecure mode:
 
 ```bash
 $IPDK_RECIPE/install/sbin/infrap4d  -grpc_open_insecure_mode=true
```

Comment:
I would suggest defining an `IPDK_INSTALL` environment variable and changing the examples to, e.g., `$IPDK_INSTALL/sbin/infrap4d` instead of `$IPDK/install/sbin/infrap4d`.

The semantic component that gets replaced is the base install directory, not its parent.

This way, the command works regardless of whether the recipe is installed in `$IPDK/install` or `/opt/p4cp`.

Reply:
Sure, good idea. Will take it as a separate PR since it will touch multiple files